martin/Cargo.toml

```diff
@@ -60,8 +60,9 @@ name = "bench"
 harness = false
 
 [features]
-default = ["fonts", "mbtiles", "pmtiles", "postgres", "sprites"]
+default = ["fonts", "lambda", "mbtiles", "pmtiles", "postgres", "sprites"]
```

Comment:
do we want to group all lambda supports into one group, or will we need to use different dependencies/code-paths for aws vs gcp vs azure?

Reply:
Fortunately, only Lambda (to my knowledge) requires shenanigans like this, everything else supports regular HTTP servers.